Automattic/wp-calypso
# WooExpress: Fixes the timeframe description on the monthly view

Closes #75483 #75299

## Proposed Changes

* Fixes the timeframe description for WooExpress plans following the design proposed here: peapX7-22h-p2. We were showing different messages for the Essential and the Performance plan(See image below). This problem was introduced on #75073.
* We can't use the `usePerMonthDescription` hook found [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plan-features-2023-grid/components/billing-timeframe.tsx#L24-L89) because the way to fetch the yearly plan variant does not work for WooExpress plans.
* Fixes #75299 adding the promotion to the annual plan below the pricing when the user is viewing the monthly plan

#### Before
<img width="1231" alt="Screen Shot 2023-04-25 at 08 56 27" src="https://user-images.githubusercontent.com/1234758/234269085-234331c3-119e-45e2-8aee-aaed0c65b3ca.png">

#### After
<img width="1242" alt="Screen Shot 2023-04-25 at 08 53 27" src="https://user-images.githubusercontent.com/1234758/234268425-0527d0b6-860a-41fa-b075-f9f9c0ff51c4.png">

## Testing Instructions

* Navigate to `/setup/wooexpress` to create a new Woo Trial site.
* Navigate to `/plans/monthly/{YOUR-SITE-SLUG}
* You should `per month`, followed by `Save X% by paying annually` below the price
